lib/vanagon/project.rb

```diff
@@ -286,9 +297,48 @@ def get_preinstall_actions(pkg_state)
       end
     end
 
+    # Collects the install trigger scripts for the project for the specified packing state
+    #
+    # @param pkg_state [String] the package state we want to run the given scripts for.
+    #   Can be one or more of the 'install' or 'upgrade'
```

Comment:
Since this is a string now should probably get rid of the `or more`

Reply:
Whoops yeah, forgot to change the comment